FFIStandaloneTests/FFIStandaloneTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+
+class FFIStandaloneTests: XCTestCase {
+    
+    func testLoggerSetup() {
+        //First call will set the log level to 0/Trace. Remaining calls will have no effect.
+        for i:Int32 in 0...4 {
+            loggerSetupWithLevel(levelInt: i)
+        }
+    }
+    
+    func loggerSetupWithLevel(levelInt: Int32) {
+        let level = CoreLogLevel(levelInt)
+        print("iOS : Logger level : \(level)")
+        let ret = setup_logger(level, false);
+        XCTAssertEqual(levelInt, ret);
```

Comment:
This is a bit weird. The calls after the first have no effect, but in Rust you just return the level you passed as a parameter.